src/_pytest/_code/code.py

```diff
@@ -223,7 +225,7 @@ def statement(self) -> "Source":
         return source.getstatement(self.lineno)
 
     @property
-    def path(self) -> Union[py.path.local, str]:
```

Comment:
I think a `trivial` changelog entry might be good, in case there are plugin authors using the `code` methods that have changed.

Reply:
Right, added.